spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -19,6 +19,17 @@
     end
   end
 
+  describe '#course' do
```

Comment:
There's this system test that we can use to test the API output end-to-end:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/system/vendor_api/vendor_receives_application_spec.rb#L52

@duncanjbrown added that to avoid us having to test every attribute separately.

Reply:
Great, thanks